crates/common/benches/opa_executor.rs

```diff
@@ -7,8 +7,8 @@ use serde_json::Value;
 use tokio::runtime::Runtime;
 
 fn load_wasm_from_file() -> Result<(), OpaExecutorError> {
-    let wasm = "auth.tar.gz";
-    let entrypoint = "auth.is_authorized";
+    let wasm = "allow_transactions";
+    let entrypoint = "allow_transactions/allowed_users";
```

Comment:
I wonder why this one isn't,
```suggestion
    let entrypoint = "allow_transactions.allowed_users";
```

Reply:
You can use either.

Reply:
Aha, thank you, I'd wondered since `entrypoint.replace('.', "/"))` was removed.